addons/website/views/snippets/s_kickoff.xml

```diff
@@ -0,0 +1,16 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+
+<template id="s_kickoff" name="Kickoff">
+    <section class="s_kickoff o_cc o_cc4 parallax s_parallax_is_fixed pt232 pb88 o_colored_level" data-oe-shape-data="{'shape': 'web_editor/Origins/14_001', 'colors': {'c3': 'o-color-4', 'c4': 'rgba(255, 0, 0, 0)'}, 'flip': [], 'showOnMobile': false}">
```

Comment:
Is `o_colored_level` really necessary ?

```suggestion
    <section class="s_kickoff o_cc o_cc4 parallax s_parallax_is_fixed pt232 pb88" data-oe-shape-data="{'shape': 'web_editor/Origins/14_001', 'colors': {'c3': 'o-color-4', 'c4': 'rgba(255, 0, 0, 0)'}, 'flip': [], 'showOnMobile': false}">
```

Reply:
Doesn't seem necessary indeed, this won't affect the preview of the snippet within the modal, right ?

Reply:
The role of this class is to correctly adapt inheritance when there are several levels of preset colors (for example, a `o_cc1` column in a `o_cc5` section). Since you're only using a preset color on your section, this should be fine ;)

Reply:
Oh okay ! Thanks for the tips 😄